repos/spack_repo/builtin/packages/scitokens_cpp/package.py

```diff
@@ -45,7 +47,8 @@ class ScitokensCpp(CMakePackage):
     depends_on("openssl")
     depends_on("sqlite")
     depends_on("curl")
-    depends_on("jwt-cpp", type="build")
+    depends_on("jwt-cpp", type="build")sc
```

Comment:
Is `sc` a typo or should the whole line be removed?

Reply:
Typo. Thanks! `jwt-cpp` remains a build dependency.

```suggestion
    depends_on("jwt-cpp", type="build")
```